packages/block-library/src/tag-cloud/edit.js

```diff
@@ -49,10 +49,10 @@ function TagCloudEdit( { attributes, setAttributes } ) {
 		largestFontSize,
 	} = attributes;
 
-	const [ availableUnits ] = useSettings( 'spacing.units' );
 	const units = useCustomUnits( {
-		availableUnits: availableUnits || [ '%', 'px', 'em', 'rem' ],
+		availableUnits: [ 'px', 'em', 'rem', 'vw', 'vh', 'pt' ],
```

Comment:
Added same units as we use for [font size picker](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.tsx#L39) including PT unit. And removed use of spacing units setting for font sizes.

Reply:
It feels to me that we should always honor the spacing units and we shouldn't remove this. Having said that, since the tag cloud has this nuance with the previous default value, maybe we should just append the `pt` to the `availableUnits` in either case (spacing.units or defaults). What do you think?

Reply:
That's a good question!  Do we have a different theme.json setting for font size units? I searched a bit and I think not.. We should probably use the same values we use for setting font sizes in typography panel (size).

My point is, if we have such a setting and we were using the wrong one, we should honor it. I'll cc @MaggieCabrera if she can provide some feedback.

Reply:
I vaguely remember having a discussion about font size unit block support with @t-hamano but I can't find that issue or PR now.

BTW unit support is broken in block.json https://github.com/WordPress/gutenberg/issues/47519

Reply:
> It is not logical to me that **spacing** units are used as the **font size** unit

I completely agree with this point, but given that theme.json doesn't support **font size** units, I think we need to respect `spacing.units` for now to maintain backward compatibility.

How about keeping the current specs but adding the `pt` unit?

```javascript
// The `pt` unit is used as the default value and is therefore
// always considered an available unit.
const units = useCustomUnits( {
availableUnits: availableUnits
	? [ ...availableUnits, 'pt' ]
	: [ '%', 'px', 'em', 'rem', 'pt' ],
} );
```

Reply:
Thank you @t-hamano, Let me know If we all are agree with the above proposed solution, ill update the PR

Also, Shouldn't we consider having a unit setting for font size in `theme.json`?

Reply:
I believe this approach is the best one to maintain backward compatibility and users to reselect the `pt` unit.
allow

> Shouldn't we consider having a unit setting for font size in "theme.json"?

Of course, this is a feature improvement that should be considered in a separate PR.